### PR TITLE
MAGECLOUD-3866: Warm up functionality does not work if no store with 'default' code

### PR DESCRIPTION
**Pull request by oshmyheliuk**
https://magento2.atlassian.net/browse/MAGECLOUD-3866

### Related PR

https://github.com/magento/ece-tools/pull/560

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento-cloud-components?pullRequest=5) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento-cloud-components?pullRequest=5) before we can accept your contribution.<br/>

File: Console/Command/ConfigShowDefaultUrlCommand.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+     */
+    private $storeManager;
+
+    /**
+     * @param StoreManagerInterface $storeManager
+     */
+    public function __construct(
+        StoreManagerInterface $storeManager
+    ) {
+        parent::__construct();
```
Comment (shiftedreality) [inline Console/Command/ConfigShowDefaultUrlCommand.php]: Parent constructor should go after properties assignment after a newline

File: Console/Command/ConfigShowDefaultUrlCommand.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        $this->storeManager = $storeManager;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function configure()
+    {
+        $this->setName('config:show:default-url')
+            ->setDescription(
```
Comment (shiftedreality) [inline Console/Command/ConfigShowDefaultUrlCommand.php]: No newlines needed

File: Console/Command/ConfigShowDefaultUrlCommand.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            /** @var Store $store */
+            $store = $this->storeManager->getDefaultStoreView();
+            $output->write($store->getBaseUrl(UrlInterface::URL_TYPE_LINK, $store->isUrlSecure()));
+
+            return Cli::RETURN_SUCCESS;
+        } catch (\Exception $e) {
```
Comment (shiftedreality) [inline Console/Command/ConfigShowDefaultUrlCommand.php]: This try-catch does not provide any value, can be removed

File: Console/Command/ConfigShowDefaultUrlCommand.php [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+{
+    /**
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
+
+    /**
+     * @param StoreManagerInterface $storeManager
+     */
+    public function __construct(
```
Comment (shiftedreality) [inline Console/Command/ConfigShowDefaultUrlCommand.php]: No newlines needed

File: Console/Command/ConfigShowDefaultUrlCommand.php (hunk 1 above)
Comment (oshmyheliuk) [reply Console/Command/ConfigShowDefaultUrlCommand.php]: Why? Is this Magento code standard?

File: Console/Command/ConfigShowDefaultUrlCommand.php (hunk 4 above)
Comment (oshmyheliuk) [reply Console/Command/ConfigShowDefaultUrlCommand.php]: Fixed

File: Console/Command/ConfigShowDefaultUrlCommand.php (hunk 2 above)
Comment (oshmyheliuk) [reply Console/Command/ConfigShowDefaultUrlCommand.php]: Fixed

File: Console/Command/ConfigShowDefaultUrlCommand.php (hunk 3 above)
Comment (oshmyheliuk) [reply Console/Command/ConfigShowDefaultUrlCommand.php]: removed

File: Console/Command/ConfigShowDefaultUrlCommand.php (hunk 1 above)
Comment (shiftedreality) [reply Console/Command/ConfigShowDefaultUrlCommand.php]: Most Magento code use parent after

Comment (oshmyheliuk) [reply Console/Command/ConfigShowDefaultUrlCommand.php]: Not most, we don't have such rule.

**Pull request merged**